packages/world/src/systems/FarmingSystem.sol

```diff
@@ -34,4 +35,24 @@ contract FarmingSystem is System {
 
     ObjectType._set(farmlandEntityId, ObjectTypes.Farmland);
   }
+
+  function growSeed(Vec3 coord) external {
+    // TODO: should this function be open?
```

Comment:
Yee I think it should be open, this is similar to how nature system will be, it will be open and this is kind of the first function in the nature system, could even put it in a new system

Reply:
but do we still need proximity checks? And if we ever add incentives we would need a valid player right?

Reply:
I dont think we need proximity checks for stuff like this or really any nature system calls. it's just progressing the world by calling tick. Yea we could require a player so we can give incentives in the future